src/shogun/kernel/Kernel.cpp

```diff
@@ -1421,8 +1423,10 @@ SGMatrix<T> CKernel::get_kernel_matrix()
 	return SGMatrix<T>(result,m,n,true);
 }
 
+
 template SGMatrix<float64_t> CKernel::get_kernel_matrix<float64_t>();
```

Comment:
I would really like to get rid of this and instead do a openmp version at some point.
@lambday another one for you :)
@lisitsyn or you? :)